src/common/config_parser.cpp

```diff
@@ -687,11 +692,10 @@ void ConfigParser::addSuboptionsULR(cli::CLIWrapper& cli) {
   // clang-format on
 }
 
-void ConfigParser::expandAliases(cli::CLIWrapper& cli) {
-  YAML::Node config;
+void ConfigParser::addAliases(cli::CLIWrapper& cli) {
   // The order of aliases does matter as later options overwrite earlier
 
-  if(config_["best-deep"].as<bool>()) {
+  cli.alias("best-deep", "true", [](YAML::Node& config) {
```

Comment:
I think these should be moved to a separate file, either by moving the function, or by using an `#include`.

```
void ConfigParser::addAliases(cli::CLIWrapper& cli) {
#include "common/aliases.inc"
}
```

@frankseide Is this ugly?

Reply:
Or better have the entire function inside `common/aliases.inc`

Reply:
Roman suggested to just put the function definition into a separate aliases.cpp which is of course much better. I have done something like that in the `bert` branch.